pynet/models/vae/vae.py

```diff
@@ -119,7 +119,7 @@ def final_conv_dim(input_dim, kernels, paddings):
 
     @staticmethod
     def init_dense_layers(input_dim, hidden_dims, act_func, dropout,
-                          final_activation=True):
+                          final_activation=False):
```

Comment:
This static method is used to build other networks.
We thus can't change the default value without modifying/checking all these networks.
We can set this parameters during the fct call.
This fct as well as the init_conv_layers fct will be moved in the utils module.